frontend/src/components/Dropdown.tsx

```diff
@@ -0,0 +1,84 @@
+import { Menu, Portal, Icon } from "@chakra-ui/react";
+import React, { useState } from "react";
+import { LuChevronDown, LuChevronUp } from "react-icons/lu";
+
+export type DropdownItemProps = {
+  children?: React.ReactNode;
+  icon?: React.ReactNode;
+} & (
+  | { href: string; onClick?: () => void }
+  | { href?: undefined; onClick: () => void }
+);
+
+const DropdownItem = ({ href, onClick, children, icon }: DropdownItemProps) => {
+  const content = (
+    <>
+      {icon && <span>{icon}</span>}
+      {children}
+    </>
+  );
+
+  let element;
+  if (href) {
+    element = <a href={href}>{content}</a>;
```

Comment:
I'm pretty sure that instead of the `<a>` tag you'd want to use the [`<Link>` tag](https://nextjs.org/docs/pages/api-reference/components/link) from NextJS, cf. the previous `DropdownLinkItem` implementation